app/Livewire/Importer.php

```diff
@@ -148,6 +152,107 @@ public function updatingTypeOfImport($type)
         }
     }
 
+    public function updatedFieldMap($value, $index)
+    {
+        \Log::error("Updated (past tense!) THE FIELD MAP! HERE's...something? $index - the value it's changing to is: $value");
```

Comment:
We probably don't want to be throwing an error here, unless you want me to take the Rollbar budget out of your bonus ;)